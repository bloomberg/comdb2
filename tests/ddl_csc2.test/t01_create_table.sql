DROP TABLE IF EXISTS t4;
DROP TABLE IF EXISTS t5;

CREATE TABLE t4(x1 SMALLINT, x2 BIGINT, x3 INT, x4 LARGEINT);$$
CREATE TABLE t5(x1 SMALLFLOAT, x2 FLOAT, x3 DOUBLE);$$

SELECT sql FROM sqlite_master WHERE name = 't4';
SELECT sql FROM sqlite_master WHERE name = 't5';

DROP TABLE t4;
DROP TABLE t5;
