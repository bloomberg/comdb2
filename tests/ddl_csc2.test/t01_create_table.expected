(sql='create table "t4"("x1" smallint, "x2" largeint, "x3" int, "x4" largeint);')
(sql='create table "t5"("x1" smallfloat, "x2" smallfloat, "x3" double);')
